test/pxci.test.js

```diff
@@ -0,0 +1,49 @@
+const should = require('should');
+const { V2CredentialsIntelligenceProtocol } = require('../lib/extract_field/credentials_intelligence/V2CredentialsIntelligenceProtocol');
+
+describe('Check Credentials Intelligent v2 hashing', function() {
+    it('Should hash username', (done) => {
+        const credentialsBody = { username: 'pxUser', password: '1234' };
+
+        const protocol = new V2CredentialsIntelligenceProtocol();
+        const hashedCredentials = protocol.ProcessCredentials(credentialsBody.username, credentialsBody.password);
+
+        (hashedCredentials.username).should.equal('9620f4cab3b3a50b9cbcb9a8d01328874ec33eb6882ae31c022f6986fc516851');
+        (hashedCredentials.password).should.equal('c958c33151f273c620ec658ac4de9abd33ad7627df5d8c468224b0bae7173eb4');
+        done();
```

Comment:
No need for `done()` function -- this is only needed if you want to end the test from inside a callback. You don't need to pass any parameter into the test:

```js
it('should do something', () => {
    // ...
});
```